lib/node-sftp-s3.js

```diff
@@ -73,6 +87,14 @@ class SFTPS3Server extends EventEmitter {
     });
   }
 
+  removePasswordLogin(username) {
+    const self = this;
+    _.remove(this.passwordLogins, (p) => {
+      self._log(util.format('Removed password login for username %s', username));
```

Comment:
Please move this log message outside of this function.  It will be called for each user in the `passwordLogins`.  It would also be useful to check that the length changed to ensure the user was actually removed.